src/main/java/com/conveyal/datatools/manager/controllers/api/UserController.java

```diff
@@ -327,6 +282,140 @@ private static Object getRecentActivity(Request req, Response res) {
         return activityList;
     }
 
+    /**
+     * Create a HTTP GET request with some default headers
+     */
+    private static HttpGet makeGetRequest(String url) {
+        HttpGet request = new HttpGet(url);
+        request.addHeader("Authorization", "Bearer " + AUTH0_API_TOKEN);
+        request.setHeader("Accept-Charset", charset);
+        return request;
+    }
+
+    /**
+     * Create a HTTP POST request with some default headers
+     */
+    private static HttpPost makePostRequest(String url) {
+        HttpPost request = new HttpPost(url);
+        request.addHeader("Authorization", "Bearer " + AUTH0_API_TOKEN);
+        request.setHeader("Accept-Charset", charset);
+        request.setHeader("Content-Type", "application/json");
+        return request;
+    }
```

Comment:
It seems like there is a lot of duplicated code in `makePostRequest` and `makeGetRequest`. Also, the same headers are set in `deleteUser`. Perhaps, this should all be refactored into a single function that takes as input an `HttpEntityEnclosingRequestBase` and sets headers the Auth0-related headers called, for example, `setAuth0Headers`. It probably wouldn't hurt to set content type for all request types just to simplify things (or you could only set that header for posts). Thoughts?